introduction/architecture.md

```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
 1) Access node
 2) Data node
 
-A access node accepts all requests from the client, distributes the requests and queries
+A client connects to an access node database. It then distributes the requests and queries
 appropriately to data nodes, and aggregates the results received from the data nodes.
 Access nodes store cluster-wide information about the different data nodes as well as how
 chunks are distributed across those data nodes. Access nodes can also store non-distributed
 hypertables, as well as regular PostgreSQL tables.
 
-Data nodes store data associated with distributed hypertables. Data nodes do not store cluster-wide
-information, and otherwise look just as if they were single TimescaleDB instances. Avoid
-directly accessing local hypertables or chunks of a distributed hypertable on a data node
-as the action can fail or make the distributed hypertable inconsistent. There are
-very few reasons to ever access a data node directly, and all accesses should be done at
-the access node level.
+Data nodes store data associated with a distribute database (i.e., the
+distributed hypertables within it). Data nodes do not store
+cluster-wide information, and otherwise look just as if they were
+stand-alone TimescaleDB instances. Avoid directly accessing local
+hypertables or chunks of a distributed hypertable on a data node as
+the action can fail or make the distributed hypertable
+inconsistent. There are very few reasons to ever access a data node
+directly, and all access should be done at the access node level.
 
 It is important to note that access nodes and data nodes both run TimescaleDB, and for all intents and
-purposes, look just like a single instance of TimescaleDB from an operational perspective. However, as
-mentioned above, all accesses for a distributed hypertable should go through the access node, and
-you should not access the data node directly.
-
-### Distributed Databases
-
-A multi-node TimescaleDB cluster contains distributed databases on which distributed hypertables can be created. Distributed databases should be configured
-to have one primary access node and multiple data nodes. An access node cannot also be a data node.
-We currently only support a single primary access node that will automatically dispatch all operations on distributed hypertables to the appropriate data node(s).
-
-### Distributed Hypertables
+purposes, look just like a single instance of TimescaleDB from an operational perspective.
 
-As discussed above, distributed databases contain distributed hypertables. Distributed hypertables are essentially
-the same as regular hypertables, except that they have an additional layer of abstraction where
-access nodes must be able to logically manage distributed hypertables. Distributed hypertables
-are also composed of chunks.
+### Configuring Distributed Hypertables
 
 With distributed hypertables, we suggest partitioning the hypertables both by space and by time.
```

Comment:
Given the description below, I do not think this is a suggestion. It is how you need to partition to actually utilize the distributed nature of the distributed hypertable, so something like "to utilize the full capacity of distributed hypertable, you should partition the hypertable in both the space and time dimension."